lib/rspec/matchers/english_phrases.rb

```diff
@@ -0,0 +1,58 @@
+module RSpec
+  module Matchers
+    # Facilitates converting ruby objects to English phrases.
+    module EnglishPhrases
+      class << self
```

Comment:
Without the `class << self`, I don't think the `private` keyword will work.

```ruby
module Works
  class << self
    def z; end
    private
    def y; end
  end
end
Works.z #=> nil
Works.y # NoMethodError: private method `y' called for Z:Module

module DoesntWork
  def self.a; end
  private
  def self.b; end
end
DoesntWork.a #=> nil
DoesntWork.b #=> nil
```

Is there another way to do private module methods?